CollapseLauncher/Classes/Helper/Loading/LoadingMessageHelper.cs

```diff
@@ -0,0 +1,111 @@
+﻿using CollapseLauncher.Extension;
+using CollapseLauncher.Helper.Animation;
+using CommunityToolkit.WinUI.Animations;
+using Microsoft.UI.Text;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using System;
+using System.Collections.Generic;
+
+namespace CollapseLauncher.Helper.Loading
+{
+    internal static class LoadingMessageHelper
+    {
+        internal static MainWindow currentMainWindow = null;
+        internal static bool isLoadingProgressIndeterminate = true;
+        internal static bool isCurrentlyShow = false;
+        internal static List<RoutedEventHandler> currentActionButtonHandler = new List<RoutedEventHandler>();
+
+        internal static void Initialize()
+        {
+            if (InnerLauncherConfig.m_window is MainWindow window)
+                currentMainWindow = window;
+        }
+
+        internal static void SetMessage(string title, string subtitle)
+        {
+            if (!string.IsNullOrEmpty(title)) currentMainWindow.LoadingStatusTextTitle.Text = title;
+            if (!string.IsNullOrEmpty(subtitle)) currentMainWindow.LoadingStatusTextSubtitle.Text = subtitle;
+
+            currentMainWindow.LoadingStatusTextSeparator.Visibility = string.IsNullOrEmpty(title) || string.IsNullOrEmpty(subtitle) ? Visibility.Collapsed : Visibility.Visible;
```

Comment:
Is this missing a check for if the Title or Subtitle already has text?
As the ```SendTimeoutCancelationMessage``` call this function with a null title, it will result in this message:
![image](https://github.com/CollapseLauncher/Collapse/assets/44784408/dd0b739a-51a8-46e6-ba15-456594d176bc)
Also, the subtitle isn't wrapping.

Reply:
This overflow and logic issue should have been solved by the latest commit